diff --git a/node_modules/@graphql-inspector/core/index.mjs b/node_modules/@graphql-inspector/core/index.mjs
index 8f5395b..fe6ba41 100644
--- a/node_modules/@graphql-inspector/core/index.mjs
+++ b/node_modules/@graphql-inspector/core/index.mjs
@@ -2,7 +2,7 @@ import { parse, Kind, TypeInfo, visit, visitWithTypeInfo, GraphQLError, getNamed
 import { __awaiter } from 'tslib';
 import inspect from 'object-inspect';
 import { DepGraph } from 'dependency-graph';
-import { Parser } from 'graphql/language/parser';
+import { Parser } from 'graphql/language/parser.js';
 
 function readDocument(source) {
     const result = {
